src/commit.rs

```diff
@@ -187,7 +187,7 @@ impl<'a> Footer<'a> {
 #[derive(Debug, Clone, Copy, Eq, PartialEq, Hash)]
 #[non_exhaustive]
 pub enum FooterSeparator {
-    /// ":"
+    /// ": "
```

Comment:
We originally had this but it was changed in #18 as the [grammer](https://github.com/conventional-commits/parser) specifies the space is optional.

I raised https://github.com/conventional-commits/parser/issues/41 but this is another issue in the grammar.

Would you be up for checking how the reference implementation handles this case?

Reply:
This seems tricky...I think I might not be up to the job 😥

Reply:
Mind at least creating an issue in this repo for discussing the state of this?